src/cards/Card.js

```diff
@@ -27,6 +27,7 @@ const StyledCard = styled.section`
     margin: 0;
     padding: 10px 20px 20px;
     font-family: 'Open Sans', sans-serif;
+    position: relative;
```

Comment:
You might want to avoid `position relative` as its buggy. Maybe there is another way to solve this?